cognite/client/data_classes/contextualization.py

```diff
@@ -162,10 +178,10 @@ def wait_for_completion(self, timeout: int = None, interval: int = 1) -> None:
         start = time.time()
         while timeout is None or time.time() < start + timeout:
             self.update_status()
-            if self.status not in ["Queued", "Running"]:
+            if self.status not in [JobStatus.QUEUED, JobStatus.RUNNING]:
```

Comment:
I see that above you're using `JobStatus.NOT_FINISHED_STATUSES`. Might not be appropriate here, but letting you know since I'm not sure if it's intended.

Reply:
Currently, Entitymatching does not have other non-finished statuses than queued, running, so just to be explicit here.